index.js

```diff
@@ -1,8 +1,13 @@
 'use strict';
 
 var transpiler = require('babel-core');
-var Filter     = require('broccoli-filter');
+var fs         = require('fs');
+var path       = require('path');
 var clone      = require('clone');
+var recreaddir = require('recursive-readdir');
+var mkpath     = require('mkpath');
+var rsvp       = require('rsvp');
+var Writer     = require('broccoli-writer');
```

Comment:
broccoli-writer has no caching, i suspect this will destroy incremental build performance. (Unless i'm missing somehting)

Maybe prefer using broccoli-filters new `rebuild` API -> https://github.com/broccolijs/broccoli/blob/master/docs/new-rebuild-api.md